src/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_CategoryPickerScreen.tsx

```diff
@@ -1,27 +1,227 @@
+                        style={styles.expandedContent}
+                        isMarkdown={true}
+                      >
+                        {getTextForLanguage(
+                          u.alternativeDescriptions,
+                          language,
+                        )}
+                      </ThemeText>
+                      <Button
+                        style={styles.chooseButton}
+                        onPress={() => {
+                          updateCategory({
+                            id: u.userTypeString,
+                            userType: u.userTypeString,
+                          });
+                          navigation.navigate(
+                            'TicketAssistant_FrequencyScreen',
+                          );
+                        }}
+                        text={t(
+                          TicketAssistantTexts.categoryPicker.chooseButton,
+                        )}
+                      />
+                    </View>
+                  }
+                />
+              );
+            })}
+          </Section>
+        ) : (
+          <>
+            {selectableTravellers.map((u, index) => {
+              return (
+                <View key={index} style={styles.a11yCategoryCards}>
+                  <TouchableOpacity
+                    onPress={() => {
+                      updateCategory({
+                        id: u.userTypeString,
+                        userType: u.userTypeString,
+                      });
+                      navigation.navigate('TicketAssistant_FrequencyScreen');
+                    }}
+                    accessible={true}
+                    accessibilityHint={t(
+                      TicketAssistantTexts.categoryPicker.a11yChooseButtonHint({
+                        value: getReferenceDataName(u, language),
+                      }),
+                    )}
+                  >
+                    <View style={styles.contentContainer}>
+                      <ThemeText
+                        style={styles.a11yTitle}
+                        type={'body__primary--bold'}
+                        isMarkdown={true}
+                      >
+                        {getReferenceDataName(u, language)}
+                      </ThemeText>
+                      <ThemeText
+                        type={'body__tertiary'}
+                        style={styles.expandedContent}
+                        isMarkdown={true}
+                      >
+                        {getTextForLanguage(
+                          u.alternativeDescriptions,
+                          language,
+                        )}
+                      </ThemeText>
+                    </View>
+                  </TouchableOpacity>
+                </View>
+              );
+            })}
+          </>
+        )}
+      </ScrollView>
+    </View>
   );
 };
+
 const useThemeStyles = StyleSheet.createThemeHook((theme) => ({
+  backdrop: {
```

Comment:
It might not be part of this task but would be great to refactor this, I saw is used on many places in the same way.